sme_financing/main/apis/fund_disbursement_api.py

```diff
@@ -0,0 +1,86 @@
+from flask import request
+from flask_restx import Resource
+from flask_restx._http import HTTPStatus
+
+from ..service.fund_disbursement_service import (
+    delete_disbursement,
+    get_all_fund_disbursements,
+    get_fund_disbursement_by_id,
+    save_disbursement,
+    update_fund_disbursement,
+)
+from .dto import FundDisbursementDTO
+
+api = FundDisbursementDTO.fund_disbursement_api
+_fund_disbursement = FundDisbursementDTO.fund_disburse
+
+
+@api.route("/fund_disburse")
+class fundDisbursementResource(Resource):
+    @api.doc("List all funding disbursements")
+    @api.marshal_list_with(_fund_disbursement, envelope="data")
+    def get(self):
+        """List all fund disbursements"""
+
+        return get_all_fund_disbursements()
+
+    @api.doc("Create a new fund disbursement")
+    @api.expect(_fund_disbursement, validate=True)
+    @api.response(201, "Funding disbursement successfully created")
+    def post(self):
+        """
+        Create a new funding criteria
+        """
+        data = request.json
+        return save_disbursement(data=data)
```

Comment:
`FundDisbursement` is always linked to a `FundingProject`; it doesn't look like you have a way of getting a `funding projec`t here. 'cause given the `fund disbursement dto` you've defined, the `data` object won't contain a `funding  project` identifier. the endpoint for creating a fund disbursement should look like this `funding_projects/{funding_project_number}/fund_disbursements`. That way you get the project number from url.

Reply:
sure